test/integration/index.test.js

```diff
@@ -1,5 +1,5 @@
 
-const { Logger, transports } = require('el-logger'); // eslint-disable-line no-unused-vars
+const { Logger } = require('@mojaloop/sdk-standard-components').Logger; // eslint-disable-line no-unused-vars
```

Comment:
```suggestion
const { Logger } = require('@mojaloop/sdk-standard-components').Logger;
```
Are you sure you need to disable this? You're definitely using `Logger` further down.

Reply:
Well spotted, that would've been there in the past and I didn't scrutinise it here.